src/reducers/profile-view.js

```diff
@@ -480,11 +479,7 @@ export type SelectorsForThread = {
   getExpandedCallNodePaths: State => CallNodePath[],
   getExpandedCallNodeIndexes: State => Array<IndexIntoCallNodeTable | null>,
   getCallTree: State => CallTree.CallTree,
-  getFilteredThreadForStackChart: State => Thread,
-  getCallNodeInfoOfFilteredThreadForStackChart: State => CallNodeInfo,
-  getCallNodeMaxDepthForStackChart: State => number,
-  getStackTimingByDepthForStackChart: State => StackTiming.StackTimingByDepth,
-  getLeafCategoryStackTimingForStackChart: State => StackTiming.StackTimingByDepth,
+  getStackTimingByDepth: State => StackTiming.StackTimingByDepth,
```

Comment:
This is the only selector that survived, but it is now backed by the normal filtering pipeline.